feat(curriculum): add Build a Bookstore Page Workshop to FSD cert (#62723)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-bookstore-page/68e97fe79367ad7b5dd6c9cd.md

---
id: 68e97fe79367ad7b5dd6c9cd
title: Step 1
challengeType: 0
dashedName: step-1
demoType: onLoad
---

# --description--

In this workshop, you will build a bookstore page by creating book cards that display information about different books. You'll learn about organizing content using `div` elements, classes, and ids.

> Start by adding an `<h1>` with the text `XYZ Bookstore`.

Reviewer: ```suggestion
Start by adding an `h1` element with the text `XYZ Bookstore`.
```

---
id: 68e97fe79367ad7b5dd6c9cd
title: Step 1
challengeType: 0
dashedName: step-1
demoType: onLoad
---

# --description--

> In this workshop, you will build a bookstore page by creating book cards that display information about different books. You'll learn about organizing content using divs, classes, and ids.

Reviewer 2: ```suggestion
In this workshop, you will build a bookstore page by creating book cards that display information about different books. You'll learn about organizing content using `div` elements, classes, and ids.
```

Start by adding an `<h1>` with the text:

> `XYZ Bookstore`

Reviewer 2: ```suggestion
Start by adding an `h1` with the text `XYZ Bookstore`.
```

# --hints--

You should have an `h1` element.

```js
assert.exists(document.querySelector('h1'));
```

Your `h1` element should have a closing tag. Closing tags have a `/` and the same element name: `</h1>`.

```js
assert.match(code, /<\/h1\>/);

> ```

Author: Will remove it in all the steps but If the camper knows how to open and close tags properly, they’d never encounter this failed test.

> ```

Reviewer 2: True. But we usually only have extra tests like this because there were frequent issues in the earlier projects. 
In general it is best to only add tests where absolutely necessary 👍🏾
